src/foam/u2/crunch/Style.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.crunch',
+  name: 'Style',
```

Comment:
Why is this being done?

Reply:
Why not make a reusable Card component instead? See com.google.foam.demos.u2.Border.js

Reply:
https://nanopay.atlassian.net/browse/NP-1375